tests/test_lm/test_integration/test_jsrun/test_3/cmds

```diff
@@ -0,0 +1,3 @@
+# Task needs to be assigned node 1, physical core 1, HW thread 8
```

Comment:
I am not sure this is possible: 8 HW threads = 2 cores as each core has up to 4 HWT, right?

Reply:
I wanted to mention 4 threads per process. If the comment is misleading, I'll change it.

Reply:
Do we have two processes? Is each process placed onto a separate physical core?